src/content/docs/en/guides/internationalization.mdx

```diff
@@ -0,0 +1,259 @@
+---
+title: Internationalization
+description: Learn how to use i18n routing in Astro.
+i18nReady: false
+---
+
+import FileTree from '~/components/FileTree.astro'
+import Since from '~/components/Since.astro'
+
+Astro's internationalization (i18n) features allow you to adapt your project for an international audience.
+
+
+## i18n Routing (experimental) 
+
+
+<Since v="3.5.0" />
+
+The experimental i18n router provides a set functionalities to help building localised websites. 
+
+By default, internationalization routing functionalities are all turned off and the user needs to manually opt in them.
+
+## Glossary
+
+The current page will use some specific terminology related to internationalization:
+- Locale: a code that represents a language.
+- Localized folder: a folder/directory that is named after a locale.
+
+
+
+## Basic Usage
+
+1. Enable the routing via configuration
+
+   ```js title="astro.config.mjs"
+   import { defineConfig } from "astro/config"
+   export default defineConfig({
+     experimental: {
+       i18n: {
+         defaultLocale: "en",
+         locales: ["es", "en", "fr"]
+       }
+     }
+   })
+   ```
+2. Create the folders and pages will contain the translated pages
+    <FileTree>
+    - src
+      - pages
+        - en
+          - index.astro
+        - es
+          - index.astro
+        - fr
+          - index.astro
+    </FileTree>
+
+    :::note
+    The localised folder doesn't need to be at the root of the `pages/` folder.
+    :::
+
+3. Use the virtual module [`astro:i18n`](#virtual-module-astroi18n) that will help you compute the correct localised URLs:
+
+    ```astro title="src/pages/es/index.astro"
+    ---
+    import { getRelativeLocaleUrl } from "astro:i18n";
+    
+    const gettingStartedUrl = getRelativeLocaleUrl("es", "getting-started"); // /es/getting-started
+    ---
+  
+    
+    <a href={gettingStartedUrl}>Got to getting started</a>
+    ```
+
+## Virtual module `astro:i18n`
+
+This module provides functions that can help you to create URLs using the locales.
+
+All the functions below take into account the following options, so the returned paths will change based on their values:
+
+- [`base`](/en/reference/configuration-reference/#base)
+- [`trailingSlash`](/en/reference/configuration-reference/#trailingslash)
+- [`build.format`](/en/reference/configuration-reference/#buildformat)
+- [`site`](/en/reference/configuration-reference/#site)
+
+If `i18n.routingStrategy` is equals to `'prefix-other-locales'`, when the functions will process the `i18n.defaultLocale` when passed as input, the returned URLs **won't** contain the `i18n.defaultLocale`.  
```

Comment:
What do you mean here as "functions"? This sentence doesn't sound right, but not sure what you mean there to offer a suggestion.